bson/value.go

```diff
@@ -553,6 +553,16 @@ func (v *Value) Binary() (subtype byte, data []byte) {
 	return st, b
 }
 
+// BinaryOK is the same as Binary, except it returns a boolean instead of
+// panicking.
+func (v *Value) BinaryOK() (subtype byte, data []byte, bool) {
+	if v == nil || v.offset == 0 || v.data == nil || v.data[v.start] != '\x05' {
+		return "", nil, false
```

Comment:
This also popped up as a compilation issue, the return here should be a byte, so something like `0x00`.

Reply:
Again my bad, didn't checked through. Sorry for the inconvenience.